cmake/GitVersion.cmake

```diff
@@ -26,4 +26,6 @@ else ()
     message(WARNING "Failed to get valid version information from Git")
 endif ()
 
-configure_file(${VERSION_IN_FILE} ${VERSION_FILE})
+include(${MACROSFILE})
```

Comment:
What's the value in passing the file as a variable? Why not just include(OpenMWMacros.cmake)?

Reply:
It can't find the file to include if it's done any other way.

CMake searches for includes relative to `CMAKE_SOURCE_DIR`, but when `GitVersion.cmake` is run (at least via Visual Studio), that takes a value which points to the location of `git-version.vcxproj`, which is dependent on where the user put their build directory.